chore: Bypass caching across all environments (#168)
ApollosProject/apollos-embeds: micro-service/README.md

_⚠️ Make sure to replace [INSERT_CHURCH_SLUG_HERE] with your church's unique identifier, or 'slug'._

### Disabling Caching for Local Testing

> For local development and testing purposes, you might want to disable caching to ensure you're receiving the latest responses directly from the API. To do this, please refer to the Apollo client configuration file:

Reviewer: ```suggestion
For local development and testing purposes, you might want to enable caching to ensure you're not receiving the latest responses directly from the API. To do this, please refer to the Apollo client configuration file:
```

[../packages/web-shared/client/apollosApiLink.js](../packages/web-shared/client/apollosApiLink.js)

> In this file, locate the header configuration within the `apollosApiLink` function and uncomment the following line:

Reviewer: ```suggestion
In this file, locate the header configuration within the `apollosApiLink` function and comment the following line:
```

```javascript

> // 'x-cache-me-not': 1,

Reviewer: ```suggestion
'x-cache-me-not': 1,
```

> 

Author: ```suggestion
```

```
